liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -114,8 +113,8 @@ private void installProject() throws Exception {
                 installApp(project.getArtifact());
             }
         } else {
-            // TODO: revise and put the message in MvnMessgaes.properties.
-            throw new MojoExecutionException("Application type is not supported," + project.getPackaging());
+            throw new MojoExecutionException(MessageFormat.format(messages.getString("error.application.not.supported"),
```

Comment:
I don't see the messages error.application.not.supported or info.loose.application.not.supported defined in MvnMessages.properties.  Also, I don't see a difference when comparing the old and new files.